packages/app-manager/src/pages/AppManagerContainer.vue

```diff
@@ -41,8 +41,10 @@
           <div class="col-12">
             <div class="d-flex justify-content-center p-4">
               <p class="text-muted">
-                <small>Zip should contain <em>index.html</em> and <em>config.json</em>, consult <a target="_blank"
-                                                                                                   href="https://molgenis.gitbooks.io/molgenis/content/user_documentation/admin-features/guide-app-manager.html">manual</a>
+                <small>Zip should contain <em>index.html</em> and <em>config.json</em>, consult
+                  <a target="_blank" 
+                    rel="noopener noreferrer" 
+                    href="https://molgenis.gitbooks.io/molgenis/content/user_documentation/admin-features/guide-app-manager.html">the manual</a>
```

Comment:
href should be: https://molgenis.gitbooks.io/molgenis/content/guide-app-manager.html